corehq/apps/userreports/pillow.py

```diff
@@ -45,6 +46,7 @@ def timed(*args, **kw):
                 table.config._id, doc['_id'], seconds
             )
             pillow_logging.warning(message)
+            _slow_ucr_assert(seconds < 5, message)
```

Comment:
planning on reducing this eventually, but just want to get a feel for how many are longer than this